app/controllers/job_invocations_controller.rb

```diff
@@ -87,7 +88,13 @@ def action_permission
 
   def prepare_composer
     if params[:feature].present?
-      JobInvocationComposer.for_feature(params[:feature], params[:host_ids], {})
+      inputs = params[:inputs].permit!.to_hash if params.include?(:inputs)
+      inputs ||= nil
```

Comment:
I think this is redundant, inputs is already registered for ruby even if the `if` condition is not evaluated to true, but if you want to keep it for readability, I think it would be better to move it one line up and simplify it to `inputs = nil`